utils/helpers.js

```diff
@@ -38,4 +40,61 @@ const readPackFile = (filePath) => path.extname(filePath) === '.json' ? readJson
  */
 const removeCWDPrefix = (filePath) => filePath.split(`${process.cwd()}/`)[1];
 
-module.exports = { readYamlFile, readJsonFile, readPackFile, removeCWDPrefix };
+/** 
+* Checks if a path is a direectory
+* @param {string} dir - The path to check
+* @returns {boolean} Whether path is a directory or not
+*/
+const isDirectory = (dir) =>
+  fs.statSync(dir).isDirectory();
+
+/** 
+* Counts the number of image files in a folder
+* @param {string} folder - The folder to count the image files from
+* @returns {number} The number of image type files in the folder
+*/
+const getImageCount = (folder) => {
+  return [...glob.sync(path.resolve(folder, '**/*'))]
+    .filter(file => isImage(file)).length
+}
+
+
+/** 
+* Gets the size of a file in Bytes
+* @param {string} file - The file to get the size of
+* @returns {Array} The file size in Bytes 
+*/
+const getFileSize = (file) => {
+  return fs.statSync(file)['size']
+}
+
+/** 
+* Parses the file extension type from a file path
+* @param {string} file - The file path to parse an extension from
+* @returns {string} The extension of the file
+*/
+const getFileExtension = (file) => {
+  return path.extname(file)
+}
+
+/** 
+* Gets an array of all files and directories in a path
+* @param {string} dir - The directory to parse, set by the BASE_PATH variable
+* @returns {Array} An array of pathnames of a globbed directory
+*/
+const globFiles = (dir) => {
+  const globbedFiles = [...glob.sync(path.resolve(dir, '**/*'))]
```

Comment:
looks like this is spreading the array & putting in another. can you simplify to returning glob.sync?